js/ui/drop_down_button.js

```diff
@@ -588,10 +597,8 @@ const DropDownButton = Widget.inherit({
                 this._createSelectionChangedAction();
                 break;
             case 'deferRendering':
-                if(!value && !this._popup) {
-                    this._renderPopup();
-                    this._renderContent();
-                }
+                this._renderPopup();
```

Comment:
Do we need to render popup if it is already rendered? To answer this question, let's cover this case by a separate test.

Reply:
fixed it